corehq/apps/accounting/async_handlers.py

```diff
@@ -250,6 +250,9 @@ class Select2CustomerInvoiceTriggerHandler(BaseSelect2AsyncHandler):
     @property
     def customer_account_response(self):
         accounts = BillingAccount.objects.filter(is_customer_billing_account=True).values_list('name', flat=True)
+        if self. search_string:
+            search_string = self.search_string.lower()
+            accounts = [x for x in accounts if x.lower().startswith(search_string)]
```

Comment:
Slightly surprised this is `startswith` instead of `in`, is that deliberate?

Reply:
Good thinking. I'm also going to change this in `Select2InvoiceTriggerHandler`